native/src/utils/getNavigationDeepLinks.ts

```diff
@@ -1,13 +1,22 @@
-import type { Position } from 'geojson'
 import { Platform } from 'react-native'
 
+import { LocationModel } from 'api-client/src'
+
 /**
  * Depending on the platform this function returns the correct deeplink for navigation app
  * @param address{string}: location address
- * @param coordinates{number[]}: latitude,longitude
+ * @param location{LocationModel}: location model with coordinates
  */
-export const getNavigationDeepLinks = (address: string, coordinates: Position): string =>
-  Platform.select({
-    ios: `maps:${coordinates[1]},${coordinates[0]}?q=${address}`,
-    android: `geo:${coordinates[1]},${coordinates[0]}?q=${address}`
+export const getNavigationDeepLinks = (address: string, location: LocationModel): string | undefined => {
```

Comment:
Please remove param `address`, it is the same as `location.location`. We also don't use js doc so you can remove the comment above completely